controller/src/test/java/io/pravega/controller/server/health/RetentionServiceHealthContributorTest.java

```diff
@@ -78,4 +95,16 @@ public void testHealthCheck() throws Exception {
         status = contributor.doHealthCheck(builder);
         Assert.assertEquals(Status.DOWN, status);
     }
+
+    private CuratorFramework getClient() throws Exception {
```

Comment:
This piece of code is duplicated in `WatermarkingServiceHealthContibutorTest.java`, could that be avoided? Reusing code is always good.